misk/web/tabs/loader/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Logs
+logs
```

Comment:
though I guess names like this are tricker

Reply:
I'll find a way to make it work, I don't have many logs outputting them right now.